tests/driver_bmx055/app.config.test

```diff
@@ -0,0 +1,13 @@
+# this file enables modules defined in Kconfig. Do not use this file for
+# application configuration. This is only needed during migration.
+# include and auto-initialize all available sensors
+CONFIG_MODULE_PHYDAT=y
+CONFIG_MODULE_SAUL=y
+CONFIG_MODULE_SAUL_REG=y
+CONFIG_MODULE_SAUL_DEFAULT=y
```

Comment:
When I use `menuconfig` I can see that `CONFIG_MODULE_SAUL_GPIO` is selected but I cant find where it is selected in the dependencies.

Reply:
Ahh dumb me: `default y if HAVE_SAUL_GPIO`